### ISSUE TITLE
'Authored by' field data - contrast change for AA compliance

### ISSUE THREAD

**Issue by Dmelady** [brharp/hjckrrh#160]
Contrast - adjustment for our theme
![image](https://cloud.githubusercontent.com/assets/13487599/9335140/55724be8-45a0-11e5-9195-7eceeb2618d7.png)
![image](https://cloud.githubusercontent.com/assets/13487599/9335145/5ba1c5a2-45a0-11e5-9365-8e39e9a9d365.png)

**Comment by mmafe** [comment]
Sorry - not able to find the URL associated with this issue. Could you please post the URL?

**Comment by jpinsent** [comment]
I think I found the pages that go with these screencaps.

The first screencap is the feature listing filtered by category: https://aoda.web.uoguelph.ca/oac/feature-category/videos
The second screencap is from a video detail page:
https://aoda.web.uoguelph.ca/oac/video/bachelor-commerce-student-profile

I don't think either of these pages can be accessed without directly typing in the URL. Joe and I avoided testing the video detail page since the pages are not intended to be viewed by users, but a feature filtered by category view may be useful to clients at some point.

It looks like the colour of the font is #777777, which falls just under the contrast requirement. Changing it to #767676 would resolve this issue.
